docs/plugin-transform-remove-undefined.md

```diff
@@ -54,12 +54,12 @@ babel --plugins babel-plugin-transform-remove-undefined script.js
 
 ```javascript
 require("@babel/core").transformSync("code", {
-  plugins: ["babel-plugin-transform-remove-undefined"]
+  plugins: ["babel-plugin-transform-remove-undefined"],
 });
 ```
 
 ## Options
 
-+ `tdz` - Detect usages before declaration/initialization in let/const(throws) and var(void 0)
+- `tdz` - Detect usages before declaration/initialization in let/const(throws) and var(void 0)
```

Comment:
It took me too long to realize why the deleted line is marked with `+` in the git diff and the new line with `-` :joy: